src/js/profile/chart.js

```diff
@@ -78,7 +89,7 @@ export class Chart extends Observable {
             })
         } else {
             chart.xAxisFormatter((d) => {
-                return numFmtAlt(d);
```

Comment:
I think that d3format should be abstracted so that we can switch formatting engines without modifying the code. Also, using a numeric index to access the format seems error-prone. Shouldn't we rather use a dictionary key which would be self-documenting?